content/docs/command-reference/params/index.md

```diff
@@ -18,44 +18,44 @@ positional arguments:
 
 In order to track parameters and hyperparameters associated to machine learning
 experiments in <abbr>DVC projects</abbr>, DVC provides a different type of
-dependencies: _parameters_. Parameters are defined using the the `-p`
-(`--params`) option of `dvc run`, using simple names like `epochs`,
-`learning-rate`, `batch_size`, etc.
+dependencies: _parameters_. They are defined in the `params` field of `dvc.yaml`
+(e.g. with the the `-p` (`--params`) option of `dvc run`). Params usually have
```

Comment:
I guess my point here is that this sentence is way more valuable than anything else around- it is something that immediately make it easier to understand what Parameters we talk about.

Reply:
OK. This made me realize we should separate the part about how they are defined in dvc.yaml from this first p so I started doing that and ended up moving and rewriting a lot of the text again... I also introduced some structure and inner anchor links to avoid repetition. PTAL